maquetado/scss/footer.scss

```diff
@@ -22,3 +22,22 @@
   font-weight: 100;
   margin: 20px 30px;
 }
+
+
+@media only screen and (max-width: 1024px) {
+  .text-copyright {
+    align-self: center;
+    margin: 50px 0;
+  }
+
+  .footer-img {
+    height: 70px;
```

Comment:
I think that isn't necessary set `height` with the property `width` is enough

Reply:
ok!